housekeeper/store/api/find.py

```diff
@@ -24,36 +26,49 @@ def bundles(self):
         LOG.info("Fetching all bundles")
         return self.Bundle.query
 
-    def bundle(self, name: str = None, bundle_id: int = None) -> models.Bundle:
+    def _get_bundle_query(self) -> Query:
+        """Return bundle query."""
+        return self.Bundle.query
+
+    def bundle(self, name: str = None, bundle_id: int = None) -> Bundle:
         """Fetch a bundle from the store."""
         if bundle_id:
             LOG.info("Fetching bundle with id: %s", bundle_id)
```

Comment:
Could you refactor these to f-strings? 😊 
```suggestion
            LOG.info(f"Fetching bundle with id: {bundle_id}")
```